setup.py

```diff
@@ -21,17 +21,22 @@
       author_email='support@irods.org',
       description='A python API for iRODS',
       long_description=long_description,
-      long_description_content_type='text/x-rst',
+      long_description_content_type='text/markdown',
       license='BSD',
       url='https://github.com/irods/python-irodsclient',
       keywords='irods',
       classifiers=[
                    'License :: OSI Approved :: BSD License',
-                   'Development Status :: 4 - Beta',
+                   'Development Status :: 5 - Production/Stable',
                    'Programming Language :: Python :: 2.7',
                    'Programming Language :: Python :: 3.4',
                    'Programming Language :: Python :: 3.5',
                    'Programming Language :: Python :: 3.6',
+                   'Programming Language :: Python :: 3.7',
+                   'Programming Language :: Python :: 3.8',
+                   'Programming Language :: Python :: 3.9',
+                   'Programming Language :: Python :: 3.10',
+                   'Programming Language :: Python :: 3.11',
```

Comment:
can put these into a different PR as well - in a run-up to release and version bump.

removing from here...